src/vmspawn/vmspawn.c

```diff
@@ -534,6 +554,10 @@ static int parse_argv(int argc, char *argv[]) {
                 arg_settings_mask |= SETTING_START_MODE;
         }
 
+        r = parse_environment();
```

Comment:
Currently, only the mac address is parsed and no relevant argument for this. So, that's not a big matter, but we should parse environment variable first, then parse arguments. So, please call this before `parse_argv()`.